wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,343 @@
+
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
+- `osu!direct`
+- `osu!keyboard`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!tablet`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
+
+Other brandings that are not covered by the osu! official branding must use a space. Examples include:
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
+
+User titles that include `osu!` as part of the title name must be capitalised. Examples include:
+
+- `osu! Alumni`
+- `osu! Champion`
+
+---
+
+`osu!` should not end the sentence. If it does however, use a full stop (`.`) (or question mark (`?`), if appropriate) to get `osu!.` (or `osu!?`).
+
+`osu!` must not be followed by an exclamation mark (`!`) (i.e. `osu!!`). It must be rewritten to ensure that this will not occur.
+
+## Terminology
+
+These words must be spelt as follows (spacing must match):
+
+- `approach circle`
+- `game mode` (or `mode`)
+- `game mod` (or `mod`)
+- `gameplay`
+- `hit burst`
+- `hit circle`
+- `hitsound`
+- `in-game`
+- `playstyle`
+- `slider tick`
+- `slider ball`
+- `slider path`
+- `Kudosu` (to be treated as a proper noun)
+
+---
+
+Some words have variants. Their preferred spelling must be used and is as follows:
+
+- `beatmap` instead of `map`.
+- `creator` instead of `beatmapper` or `mapper`.
+- `mapped` instead of `beatmapped`.
+- `BN` or `Beatmap Nominators` when referring to the *Beatmap Nominators*.
+- `sign in` instead of `log in`, unless the name of a button or link says otherwise.
+- `sign out` instead of `log out`, unless the name of a button or link says otherwise.
+- `register` instead of `sign up`, unless the name of a button or link says otherwise.
+
+### Tournament articles
+
+These words must be spelt as follows (spacing must match):
+
+- `NoMods`, `NoMod`, or `NM`
+- `FreeMods`, `FreeMod`, or `FM`
+- `Tiebreaker`
+
+## Abbreviations, acronyms, and initialisms
+
+Abbreviations, acronyms, and initialisms must have their meaning written out upon their first occurrence. Other occurrences are optional, but done only when necessary. The following is an example:
+
+```markdown
+The NC (Nightcore) mod is similar to the DT (Double Time) mod because both NC and DT increase the speed of the music by 50%. However, NC will change the pitch of the music and add a clap and finish to the beat.
+```
+
+Abbreviations, acronyms, and initialisms must be capitalised, with exception. The following are examples:
+
+- `CS` for `Circle Size`
+- `AR` for `Approach Rate`
+- `DT` for `Double Time`
+- `SBS` for `Storyboard Scripting`
+
+---
+
+Use `e.g.` for "for example" and `i.e.` for "that is".
+
+---
+
+Abbreviations, acronyms, and initialisms should not be pluralised. The following is an example:
+
+```markdown
+Avoid:  BNs can nominate your beatmapset.
+
+Prefer: Beatmap Nominators (BN) can nominate your beatmapset.
```

Comment:
does the order matter here? I assume it doesn't, otherwise the example contradicts with the previous one, *The NC (Nightcore) mod ... .*

Reply:
I don't think it does, but I'll change it to match the previous. This may be something that'll need some discussion.

Reply:
My mistake, I flipped it to let Beatmap Nominators be plural to avoid having the initialism being plural. I'll just change the example to omit it instead.